buddyboss/buddyboss-platform
# Fixed the issue with assigning profile type to user when sending multiple member invites

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #1634.

### How to test the changes in this Pull Request:

1. Create profile types.
2. Allow email invitation.
3. Go to the Email Invite page.
4. Invite multiple users at once and assigned them a profile type.
5. Try to accept all the invitations and notice that only the first invited user will have the profile type.

### Proof Screenshots or Video

https://www.loom.com/share/654e8e4e59964efba0101a5300027898

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
